apps/web/app/(navigation)/Favourite.tsx

```diff
@@ -0,0 +1,85 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import { usePathname } from "next/navigation";
+import { useSession } from "next-auth/react";
+import { env } from "env.mjs";
+import { BudgetType } from "../../services/mutations";
+import { Icon } from "ui";
+import { useTranslate } from "lib/hooks";
+
+//props
+interface FavouriteProps {
+  ariaLabel?: string;
+  budgets: BudgetType[];
+  isFav: boolean;
+  budgetId: string | number;
+  activeHref: string;
+}
+
+type Active = {
+  active: boolean;
+};
+
+const Wrapper = styled.div<Active>`
```

Comment:
this should button, because right now it is not accessible with the keyboard

```suggestion
const Wrapper = styled.button<Active>`
```
Once you change it to `button` then you can access it and switch on/off with keyboard. There is some additional styling needed to get rid of default button styles.

https://github.com/intive/patronage2023-js/assets/80450315/60fe06f3-a05a-480c-845f-24ed29fa7100

Reply:
@wyola thank you for your comment, I changed that